cmd/blockio/main.go

```diff
@@ -55,6 +56,10 @@ func error(format string, args ...interface{}) {
 func main() {
 	// Parse commandline arguments
 	flag.Usage = usage
+	flag.Func("prefix", "set mount prefix for system directories", func(s string) error {
```

Comment:
This looks an odd failure. Maybe updating the version of 'golangci-lint' would fix this...

Reply:
This was caused by function named `error()` above shadowing the built-in type. Took actually a while for me to understand what on earth was going on there 🤪 Now fixed